src/types/markers.js

```diff
@@ -330,10 +330,14 @@ type GCMajorCompleted_Shared = {|
   // 'None' as a reason.
   nonincremental_reason?: 'None' | string,
 
-  // The allocated space for the whole heap before the GC started.
+  // The total size of GC things before and after the GC.
   allocated_bytes: number,
   post_heap_size?: number,
 
+  // The total size of malloc data owned by GC things before and after the GC.
+  pre_malloc_heap_size: number,
+  post_malloc_heap_size: number,
```

Comment:
If it can be missing, it would be good that the types tell that; you can specify that it's optional by using `?:`:

```suggestion
  pre_malloc_heap_size?: number,
  post_malloc_heap_size?: number,
```